verilog/formatting/tree_unwrapper.cc

```diff
@@ -760,8 +764,11 @@ void TreeUnwrapper::SetIndentationsAndCreatePartitions(
       const int indent = DirectParentIsFlowControlConstruct(Context())
                              ? style_.indentation_spaces
                              : 0;
-      VisitIndentedSection(node, indent,
-                           PartitionPolicyEnum::kFitOnLineElseExpand);
+      // Apply kAppendFittingSubPartitions only to kMacroCall
+      const auto partition_policy = (tag == NodeEnum::kMacroCall)
```

Comment:
Rather than add the conditional on tag, could we separate `kMacroCall` into its own case?
And would it make sense to handle `kFunctionCall` in the same group, for consistency?

Reply:
I'll move this onto it's own case.

It would make sense but it might need some preparation, for example create more generic version of that https://github.com/google/verible/blob/master/verilog/formatting/tree_unwrapper.cc#L1267

Reply:
It might take more work to make that more generic/consistent/reusable.  We can focus on macro calls for now.